include/IGPUBuffer.h

```diff
@@ -18,7 +19,7 @@ namespace video
 /** For additional OpenGL DSA state-free operations such as flushing mapped ranges or
 buffer to buffer copies, one needs a command buffer in Vulkan as these operations are
 performed by the GPU and not wholly by the driver, so look for them in IDriver and IVideoDriver. */
-class IGPUBuffer : public core::IBuffer, public IDriverMemoryBacked
+class IGPUBuffer : public core::IBuffer, public IDriverMemoryBacked, public asset::IDescriptor

```

Comment:
wrong chain of inheritance, make a `class asset::IBuffer : public core::IBuffer, public asset::IDescriptor {};`

Reply:
@Crisspl ?

Reply:
no idea why, but will do some time

Reply:
@Crisspl TODO.

Reply:
https://github.com/buildaworldnet/IrrlichtBAW/pull/358#discussion_r326556799